pkg/imageregistry/imageregistry.go

```diff
@@ -134,8 +134,23 @@ func (builder *Builder) GetManagementState() (v1.ManagementState, error) {
 	return builder.Object.Spec.ManagementState, nil
 }
 
+// GetStorageConfig fetches imageRegistry Storage configuration.
+func (builder *Builder) GetStorageConfig() (imageregistryv1.ImageRegistryConfigStorage, error) {
+	if valid, err := builder.validate(); !valid {
+		return imageregistryv1.ImageRegistryConfigStorage{}, err
+	}
+
+	glog.V(100).Infof("Getting imageRegistry Storage configuration")
+
+	if !builder.Exists() {
+		return imageregistryv1.ImageRegistryConfigStorage{}, fmt.Errorf("imageRegistry object doesn't exist")
```

Comment:
return nil, fmt.Errorf("imageRegistry object doesn't exist")

Reply:
I prefer to leave it like this